pkg/rehearse/jobs.go

```diff
@@ -337,10 +341,56 @@ func (jc *JobConfigurer) ConfigurePresubmitRehearsals(presubmits config.Presubmi
 	return rehearsals
 }
 
+func resolveInlineCiOpConfig(container v1.Container, resolver registry.Resolver, loggers Loggers) error {
```

Comment:
I was hoping we could DRY this out with `configFromResolver` -- some abstraction around `func(org, repo, branch string) ReleaseBuildConfiguration` with either a local resolution or HTTP request as a backing

Reply:
I think that would be very messy, because you would then need to provide a resolver and either a map of configs or path to configs for local resolution, all of which you don't have when doing remote resolution. The handling of the 2 cases shares no common code, so it would be unreasonable to handle both in the same function IMO.
There is a lot of shared resolution code in the confgresolver's `main.go` and in the `jobs.go` files though. I'll DRY those functions out.

Reply:
Hmm. Could we improve that with factoring and better interfaces? The problem we're solving here in rehearsal is the same as that we have in the registry, no? We want to take a configuration from the repo and return the fully resolved data. Sure, in the registry we first get the org/repo/branch as parameters but everything downstream of that is the same, right? I think the tricky bit with rehearsals is getting the tool to have as little bespoke logic as possible. Almost all bugs we've had in rehearsal are places where we do something here, and only here. If the data transformation is the same (roughly `func(data userConfig) (resolvedConfig, error)`) I would love it to be shared.